main.go

```diff
@@ -446,6 +448,15 @@ func main() {
 	prometheusConfigF := kingpin.Flag("prometheus-config", "Prometheus configuration file path").Default("/etc/prometheus.yml").String()
 	prometheusURLF := kingpin.Flag("prometheus-url", "Prometheus base URL").Default("http://127.0.0.1:9090/prometheus/").String()
 
+	victoriaMetricsURLF := kingpin.Flag("victoriametrics-url", "VictoriaMetrics base URL").
+		Default("http://127.0.0.1:8428/").String()
+	victoriaMetricsVMAlertURLF := kingpin.Flag("victoriametrics-vmalert-url", "VictoriaMetrics VMAlert base URL").
+		Default("http://127.0.0.1:8880/").String()
+	victoriaMetricsConfigF := kingpin.Flag("victoriametrics-config", "VictoriaMetrics scape configuration file path").
+		Default("/etc/victoriametrics-promscrape.yml").String()
+	victoriaMetricsBaseConfigF := kingpin.Flag("victoriametrics-baseconfig", "VictoriaMetrics basic configuration file, can be mounted to container").
```

Comment:
could you please expand the description and define the purpose of the flags for base config and config

Reply:
Probably this flag isn't needed? Probably it would be better just exporting `BaseConfigPath` const from `services/prometheus` package and passing it to VictoriaMetrics packages?

Reply:
Removed flag.